app/src/main/res/layout/content_event.xml

```diff
@@ -555,14 +555,15 @@
                     android:visibility="gone"
                     tools:visibility="visible"/>
 
-                <Button
+                <com.google.android.material.button.MaterialButton
                     android:id="@+id/feedbackBtn"
+                    style="@style/ThreeDS2TextButtonStyle"
```

Comment:
What's this?

Reply:
For UI.

Reply:
What does it do which is not in default style?

Reply:
It shows elevation. Even if a transparent background is set.

Reply:
Merging this. But I see no documentation for this

Reply:
Found it. It is from the stripe library.

Reply:
Wow. Please revert it in the next development iteration ASAP.

Reply:
> Wow. Please revert it in the next development iteration ASAP.

Sure :+1: